### PR TITLE
build: bundle size improvements

### PR DESCRIPTION
**Pull request by eszthoff**
Story: [ONEUI-75](https://jira.textkernel.nl/browse/ONEUI-75)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,21 +84,18 @@ OneUI comes with a Storybook of examples for all components. [Click here](https:
...
+
 2. Within the OneUI root, run `npm run storybook`
 3. Go to http://localhost:9001 to check out examples of all OneUI components
 
-## Example of implementation
+## Example implementation
 
 An implementation example can be found in the example directory, along with instructions on how to run it.
 
-## Testing
```
Comment (carlobernardini) [inline README.md]: Is this no longer relevant? I can't see how OneUI tests are to be ran (apart from testing the application that implements OneUI)

Comment (eszthoff) [reply README.md]: This section is not relevant for README but for CONTRIBUTE. However just to list the scripts that are in package.json is quite redundant. If you think it has actual value, I can put it in CONTRIBUTE.md#Testing

Comment (carlobernardini) [reply README.md]: Ok agree, if you want you can still add it to the paragraph you mention.

File: README.md [hunk 2]
```diff
@@ -110,11 +106,29 @@ All CSS variables (‘[custom properties](https://developer.mozilla.org/en-US/do
...
+    '@textkernel/oneui',
+    // other libraries you might need to recompile
+].join('|');
+
+const config = {
+    transformIgnorePatterns: [`[\\\/]node_modules[\\\/](?!(${librariesToRecompile})).*$`],
+};
+```
+
+The above RegEx will tell jest to exclude all node_modules (default behavior) but the ones listed in the array above.
```
Comment (mukiienko) [inline README.md]: Could you please write jest =-> Jest

File: README.md [hunk 3]
```diff
@@ -110,11 +106,29 @@ All CSS variables (‘[custom properties](https://developer.mozilla.org/en-US/do
...
+
+```js
+// Jest configuration to recompile libraries
+const librariesToRecompile = [
+    '@textkernel/oneui',
+    // other libraries you might need to recompile
+].join('|');
+
+const config = {
+    transformIgnorePatterns: [`[\\\/]node_modules[\\\/](?!(${librariesToRecompile})).*$`],
```
Comment (mukiienko) [inline README.md]: Did you check this expression?  For me, it looks weird

**Pull request merged**